style.css

```diff
@@ -347,3 +353,41 @@ hr {
   color: #fff;
   background-color: #3347ff;
 }
+
+.enlarge {
+  all: unset;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  padding-left: 48px;
+  padding-top: 120px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 40px;
+  font-size: 56px;
+  font-weight: 700;
+  color: #fff;
+  list-style-type: none;
+  background: #6070ff;
+  opacity: 70%;
+  mix-blend-mode: multiply;
```

Comment:
- [ ] Great job implementing the functionalities 👏 . Please make sure your design looks as close as possible to the Figma design. Also, try increasing the opacity 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/179811572-23cac498-356f-4849-80a7-079122041177.png)  |  ![image](https://user-images.githubusercontent.com/74814780/179811599-f8edd7a7-4eab-4d0f-a9bd-841bca4648e9.png)